api/HTMLElement.json

```diff
@@ -597,6 +597,56 @@
           }
         }
       },
+      "auxclick_event": {
```

Comment:
Can you move these entries to Element.json and remove the other additions in this PR?